src/components/minor-button/minor-button.spec.tsx

```diff
@@ -14,6 +14,14 @@ const render = (props: ButtonProps, renderer: any = shallow) => {
   return renderer(<MinorButton {...props} />);
 };
 
+describe("when no props other than children are passed into the component", () => {
+  it("renders the default props and children", () => {
+    const wrapper = render({ children: "foo" });
```

Comment:
```suggestion
    const wrapper = mount(<MinorButton>foo</MinorButton>);
```
If you are just rendering the button with a child here, you can do this more simply using `mount`. This would also allow you to remove the `render` function entirely.

Reply:
I think it's okay to use the render function for now, it will likely mean the refactor to RTL will be slightly more straightforward as we will be using render functions etc

Reply:
Good point 👍🏼 In that case @DobroTora, could you please amend the `render` function's `renderer` parameter to remove the `any` type and replace it with `typeof shallow`? As we are trying to avoid the use of `any` in TypeScript where possible as it means type checks are bypassed.